vnext/Common/Common.vcxproj

```diff
@@ -43,22 +43,6 @@
       <Configuration>Release</Configuration>
       <Platform>ARM64</Platform>
     </ProjectConfiguration>
-    <ProjectConfiguration Include="WinUI3|ARM">
-      <Configuration>WinUI3</Configuration>
-      <Platform>ARM</Platform>
-    </ProjectConfiguration>
-    <ProjectConfiguration Include="WinUI3|ARM64">
-      <Configuration>WinUI3</Configuration>
-      <Platform>ARM64</Platform>
-    </ProjectConfiguration>
-    <ProjectConfiguration Include="WinUI3|Win32">
-      <Configuration>WinUI3</Configuration>
-      <Platform>Win32</Platform>
-    </ProjectConfiguration>
-    <ProjectConfiguration Include="WinUI3|x64">
-      <Configuration>WinUI3</Configuration>
-      <Platform>x64</Platform>
-    </ProjectConfiguration>
```

Comment:
Can't remember, but is this an artifact of WinUI3 previously being a build config instead of a prop?

Reply:
Correct.

This entry is outdated, and could lead to an incorrect VS solution matrix.